client/lib/directly/index.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * @file Interface to the third-party Real Time Messaging (RTM) widget from Directly.
+ *
+ * @see ./README.md for a higher-level overview
+ * @see https://cloudup.com/cySVQ9R_O6S for Directly's configuration guide
+ */
+
+/**
+ * External dependencies
+ */
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import { loadScript } from 'lib/load-script';
+
+const DIRECTLY_RTM_SCRIPT_URL = 'https://widgets.wp.com/directly/embed.js';
+const DIRECTLY_ASSETS_BASE_URL = 'https://www.directly.com';
+let directlyPromise;
+
+/**
+ * Gets the default set of options to configure the Directly RTM widget.
+ * It's important to keep this config in a getter function, rather than a constant
+ * on the module scope, to prevent import-time errors errors that could crash Calypso.
+ *
+ * @see https://cloudup.com/cySVQ9R_O6S for all configuration options
+ *
+ * @returns {Object} The default configuration options
+ */
+function getDefaultOptions() {
+	const ids = config( 'directly_rtm_widget_ids' );
+	const env = config( 'directly_rtm_widget_environment' );
+
+	return {
+		id: ids[ env ],
```

Comment:
Why not put different ids behind one config key?

```js
id: config( 'directly_rtm_widget_id' )
```

Config is going to perform exactly the same operation.